### ISSUE TITLE
`no-inline-html` (`MD033`) - option to allow any HTML that doesn't have a markdown syntax equivalent

### ISSUE THREAD

**Issue by DetachHead** [rvben/rumdl#722]
### Problem or motivation

instead of having to manually maintain a list of allowed or disallowed html syntax, it would be nice if i could configure the rule to only ban html tags that have a markdown equivalent

### Proposed solution

something like this:

```toml
[MD033]
allow-if-no-markdown-syntax = true
```
(probabloy not the best name but you get the idea)

### Feature category

Improve existing rule

### Example markdown

```markdown
<!-- this would not report an error because `kbd` has no markdown syntax equivalent-->
Press <kbd>F12</kbd>

<!-- this would report an error because `b` has dedicated markdown syntax (`**`) -->
foo <b>bar</b>
```

### Contribution

- [ ] I would be willing to submit a PR to implement this feature

**Comment by DetachHead** [comment]
ideally this setting would also apply to `table-allowed-elements` as well

**Comment by rvben** [comment]
Shipped in v0.2.48, as a sentinel in the existing list rather than a separate flag:

```toml
[MD033]
allowed-elements = ["no-markdown-equivalent"]
```

On your example `<kbd>` is accepted and `<b>` is reported. A sentinel rather than a boolean because it composes: `allowed-elements = ["no-markdown-equivalent", "b"]` allows both, which a separate flag could not express without a precedence rule between the two.

Your follow-up is covered too. The same sentinel works in `table-allowed-elements`, so with it set a `one<br>two` cell passes while `<b>bold</b>` in a cell is still reported. Thanks @DetachHead for raising that one before it shipped half-done.
